iina/MPVProperty.swift

```diff
@@ -1,174 +1,62 @@
 import Foundation
 
 struct MPVProperty {
-  /** audio-speed-correction */
-  static let audioSpeedCorrection = "audio-speed-correction"
-  /** video-speed-correction */
-  static let videoSpeedCorrection = "video-speed-correction"
-  /** display-sync-active */
-  static let displaySyncActive = "display-sync-active"
-  /** filename */
-  static let filename = "filename"
-  /** filename/no-ext */
-  static let filenameNoExt = "filename/no-ext"
   /** file-size */
```

Comment:
Generated source, must not be modified.

Reply:
Fixed in ce4a8bca1c16f0f7077a75b0fbe75b010cc44c95